app/controllers/main_routes/laborHistory.py

```diff
@@ -27,36 +27,42 @@
 
 @main_bp.route('/laborHistory/<id>', methods=['GET'])
 @main_bp.route('/laborHistory/<departmentName>/<id>', methods=['GET'])
-def laborhistory(id, departmentName = None):
+def laborhistory(id, departmentName=None):
     try:
         currentUser = require_login()
         if not currentUser:                    # Not logged in
             return render_template('errors/403.html'), 403
         student = getOrCreateStudentRecord(bnumber=id)
-        studentForms = FormHistory.select().join_from(FormHistory, LaborStatusForm).join_from(FormHistory, HistoryType).where(FormHistory.formID.studentSupervisee == student,
-        FormHistory.historyType.historyTypeName == "Labor Status Form").order_by(FormHistory.formID.startDate.desc())
-        authorizedForms = set(studentForms)
+        studentForms = (FormHistory.select(FormHistory, LaborStatusForm.termCode, LaborStatusForm.jobType)
+                                   .join_from(FormHistory, LaborStatusForm)
+                                   .join_from(FormHistory, HistoryType)
+                                   .where(FormHistory.formID.studentSupervisee == student, 
+                                          FormHistory.historyType.historyTypeName == "Labor Status Form"))
```

Comment:
There are slashes used to separate lines just below this, so either change this to use slashes or change the slashed lines to be wrapped in the parentheses instead.

Reply:
Good catch, I believe I've resolved the ones below to use parentheses.